core/imageroot/usr/local/nethserver/agent/actions/get-node-status/validate-input.json

```
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "get-node-status input",
+    "$id": "http://schema.nethserver.org/agent/get-node-status-input.json",
+    "description": "The action does not accept any input",
```

The action input stream content is always interpreted as a JSON data type and cannot be empty.

```suggestion
    "description": "The action expects a null value as input",
```